files/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": "0.0.0",
+  "private": true,
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "scripts": {
+    "prepare": "yarn workspace <%= addonName %> run prepare",
+    "start": "npm-run-all --parallel start:*",
```

Comment:
how much do you want to discuss things on this PR? 
are you just looking for obvious bad things? or?
Should we open issues for the little things to change?

because exploring stuff like concurrently, turborepo, yarn each-workspace, etc may be valuable

Reply:
I think unless it's totally clear what the better alternative is, and that choice is pretty much uncontroversial, we should decouple this form this PR IMHO.